ci/azure/posix.yml

```diff
@@ -29,11 +29,13 @@ jobs:
       displayName: 'Setup environment and build pandas'
 
     - script: |
-        source activate pandas-dev
-        ci/run_tests.sh
+        conda run -n pandas-dev --no-capture-output ci/run_tests.sh
```

Comment:
Why do these use `conda run` now?

Reply:
`source activate`/`source deactivate` is deprecated, this is the proper way to run stuff inside an environment now (apart from `conda activate`)

Reply:
Gotcha, thanks for the info 👍